.github/workflows/codeCoverage.yml

```diff
@@ -3,6 +3,7 @@ name: Run code coverage
 on: 
   pull_request: 
     types: [opened]
+  workflow_dispatch: {}
```

Comment:
this allows the github action to be re-triggered whenever and not just when PRs are created